release-schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "definitions": {
+    "Address": {
+      "properties": {
+        "countryCode": {
+          "title": "Country code",
+          "description": "The code of the country. An ISO 3166-1 alpha-2 code is recommended, but other codes are allowed.",
```

Comment:
Also, since we are making the codelist closed, add something like:

```md
If you need to use a user-assigned code, [create an issue](https://github.com/open-contracting/standard/issues) to discuss its addition to the codelist.
```

Reply:
Should we add the extra sentence about requesting codes to be added?

Reply:
Yes, both in the schema and the README. I'll do it.

Reply:
Done